synfig-studio/src/gui/resources/Makefile.am

```diff
@@ -2,15 +2,7 @@
 
 MAINTAINERCLEANFILES = \
 	Makefile.in
```

Comment:
Why not write like the css Makefile.am style?
https://github.com/synfig/synfig/blob/520d9721a9d28243f8d2ba2efce4c4178a8ecf9a/synfig-studio/src/gui/resources/css/Makefile.am#L3

It's up to you but if you want to write in newline then use 2 spaces to indent

Reply:
@Keyikedalube Fixed :)

Reply:
You missed this in your new commit but that's OK. Could be taken care of afterward.

Resolved :)